### PR TITLE
Feature statistic daily

### PR DESCRIPTION
**Pull request by trandaison**
@at-dungnh 
I've just finished this feature. Please help me review this one. Thank you so much!

### CODE REVIEW

File: app/Models/Bill.php [hunk 1]
```diff
@@ -42,4 +43,15 @@ public function billDetails()
...
     }
+
+    /**
+     * Get all today's bills
+     *
+     * @return Illuminate\Database\Eloquent\Collection
+     */
+    public static function getTodays()
+    {
+        return Bill::where('bills.created_at', '>=', DB::raw("concat(CURDATE(), ' 00:00:00')"))
```
Comment (dzungnguyenh) [inline app/Models/Bill.php]: [rank C] I think we need declare this as private variable for initial time.

File: app/Models/BillDetail.php [hunk 2]
```diff
@@ -43,4 +44,29 @@ public function product()
...
     }
+
+    /**
+     * Get total daily amount
+     *
+     * @return int
+     */
+    public static function dailyTotalAmount()
+    {
+        return BillDetail::where('bills_details.created_at', '>=', DB::raw("concat(CURDATE(), ' 00:00:00')"))
```
Comment (dzungnguyenh) [inline app/Models/BillDetail.php]: [rank C] wow, we are using initial time for many place, so I think we need arrange it again.

File: app/Models/BillDetail.php [hunk 3]
```diff
@@ -43,4 +44,29 @@ public function product()
...
+     * Get daily percentage of categories
+     *
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public static function getTodays()
+    {
+        return BillDetail::join('products', 'bills_details.product_id', '=', 'products.id')
+                        ->join('categories', 'products.category_id', '=', 'categories.id')
+                        ->select('categories.id', 'categories.name', DB::raw('round(sum(bills_details.amount) / ' . BillDetail::dailyTotalAmount() . ' * 100) as percentage'))
+                        ->where('bills_details.created_at', '>=', DB::raw("concat(CURDATE(), ' 00:00:00')"))
```
Comment (dzungnguyenh) [inline app/Models/BillDetail.php]: [rank C] change this too (initial time)

File: app/Models/BillDetail.php [hunk 4]
```diff
@@ -43,4 +44,29 @@ public function product()
...
+    /**
+     * Get daily percentage of categories
+     *
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public static function getTodays()
+    {
+        return BillDetail::join('products', 'bills_details.product_id', '=', 'products.id')
+                        ->join('categories', 'products.category_id', '=', 'categories.id')
+                        ->select('categories.id', 'categories.name', DB::raw('round(sum(bills_details.amount) / ' . BillDetail::dailyTotalAmount() . ' * 100) as percentage'))
```
Comment (dzungnguyenh) [inline app/Models/BillDetail.php]: [rank C] I think you can define percentage as a private variable.

**Comment by trandaison** [comment]
@at-dungnh I've fixed all issues follow your comments. Please help me review this one again. Thank you!

**Comment by dzungnguyenh** [comment]
it's seem good. Thanks.

**Pull request merged**